vsts/releaseTest.ps1

```diff
@@ -26,20 +26,20 @@ Invoke-Expression $runTestCmd
 
 $gateFailed = $LASTEXITCODE
 
-if (isWindows) 
+if (isWindows)
 {
-	Write-Host Stop ETL logging
-	logman stop IotTrace
-	logman delete IotTrace
+    Write-Host Stop ETL logging
+    logman stop IotTrace
+    logman delete IotTrace
 }
 
-if ($gateFailed) 
+if ($gateFailed)
 {
-	Write-Error "Testing was not successful, exiting..."
-	exit 1
+    Write-Error "Testing was not successful, exiting..."
+    exit 1
 }
-else 
+else
 {
-	Write-Host "Testing was successful!"
-	exit 0
+    Write-Host "Testing was successful!"
+    exit 0
 }
```

Comment:
Do we need a new line at end of this file?

Reply:
Need? No, but we generally like to have one.